### PR TITLE
Documentation for voice Recorder molecule

### PR DESCRIPTION
**Pull request by Shryansh107**
added documentation for voice Recorder molecule

### CODE REVIEW

File: docs/molecules/voiceRecorder.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+
+- **Recording Status**: Initial state of the recording (true for recording, false for idle).
+  - Default Value: false
+
+#### Error and Processing Messages
+
+You can customize the error and processing messages displayed during different states of the recorder:
+
+- **Recorder Error Message**: Message displayed when the recorded audio is not recognized or an error occurs.
+  - Default Message: "Your question was not recognized. Please try speaking more clearly."
```
Comment (prtkjakhar) [inline docs/molecules/voiceRecorder.md]: bolden this as well

File: docs/molecules/voiceRecorder.md [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+These configuration values allow you to tailor the Voice Recorder component to suit your application's requirements, providing flexibility in icon selection, audio analysis parameters, and user feedback messages.
+
+
+### Additional Customization
+For further customization, you can modify the component's styling by editing the CSS rules defined in the `index.module.css` file. Adjust styles such as font size, padding, margins, and colors to match your application's design guidelines.
+
+
+## Notes
+- The molecule simulates the voice recording process for demonstration purposes and does not handle actual audio processing.
+- The OTP length is hardcoded to 4 digits.
```
Comment (prtkjakhar) [inline docs/molecules/voiceRecorder.md]: Why is this present here in recorder documentation?

**Review by prtkjakhar — CHANGES_REQUESTED** [review]

File: docs/molecules/voiceRecorder.md (hunk 1 above)
Comment (Shryansh107) [reply docs/molecules/voiceRecorder.md]: resolved this issue

File: docs/molecules/otpPage.md [hunk 3]
```diff
@@ -8,7 +8,7 @@ sidebar_label: OTP Page
   <title> OTP Page </title>
   <meta
     name="description"
-    content="your meta content goes here"
+    content="A React component for recording audio with customizable icons and error messages. Users can start and stop recording, with visual feedback during the recording process."
```
Comment (prtkjakhar) [inline docs/molecules/otpPage.md]: why is this recorder meta inside otp page file

File: docs/molecules/voiceRecorder.md [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+title: Voice Recorder
+sidebar_label: Voice Recorder
+---
+
+
+<head>
+  <title> Voice Recorder </title>
+  <meta
+    name="description"
+    content="your meta content goes here"
```
Comment (prtkjakhar) [inline docs/molecules/voiceRecorder.md]: should have been here

File: docs/molecules/otpPage.md (hunk 3 above)
Comment (Shryansh107) [reply docs/molecules/otpPage.md]: resolved

File: docs/molecules/voiceRecorder.md (hunk 4 above)
Comment (Shryansh107) [reply docs/molecules/voiceRecorder.md]: resolved

**Pull request merged**